source/server-selection/server-selection.rst

```diff
@@ -1001,10 +1003,12 @@ If ``mode`` is 'primaryPreferred', select the primary if it is known, otherwise
 attempt the selection algorithm with ``mode`` 'secondary' and the user's
 ``maxStalenessSeconds`` and ``tag_sets``.
 
-For all read preferences modes except 'primary', clients MUST set the ``SecondaryOk``
-wire protocol flag to ensure that any suitable server can handle the request.  Clients
-MUST NOT set the ``SecondaryOk`` wire protocol flag if the read preference mode is
-'primary'.
+For all read preferences modes except 'primary', clients MUST set the
+``SecondaryOk`` wire protocol flag (OP_QUERY) or ``$readPreference`` global
+command argument (OP_MSG) to ensure that any suitable server can handle the
+request. If the read preference mode is 'primary', clients MUST NOT set the
+``SecondaryOk`` wire protocol flag (OP_QUERY) or ``$readPreference`` global
+command argument (OP_MSG).
```

Comment:
While working on this PR I realized the server selection spec is _very_ light on documentation for using the `$readPreference` global command argument. This change pertains to [DRIVERS-1918](https://jira.mongodb.org/browse/DRIVERS-1918). I opted not to revise [Passing read preference to mongos and load balancers](https://github.com/mongodb/specifications/blob/master/source/server-selection/server-selection.rst#passing-read-preference-to-mongos-and-load-balancers) in this PR, as that would be a much more intensive change.